sklearn/preprocessing/_encoders.py

```diff
@@ -652,17 +779,31 @@ def inverse_transform(self, X):
         found_unknown = {}
 
         for i in range(n_features):
-            n_categories = len(self.categories_[i])
+            if self.drop is None:
+                cats = self.categories_[i]
+            else:
+                cats = np.array([cat for idx, cat in
```

Comment:
Can't you just do `np.delete(self.categories_[i], self.drop_idx_[i])`?

Do you need to handle the drop_idx_[i] is None case here?

Reply:
Sure, that makes more sense.